common/src/main/java/com/wynntils/features/utilities/SkillPointLoadoutsFeature.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © Wynntils 2023-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.utilities;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.mc.event.ScreenOpenedEvent;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.screens.skillpointloadouts.SkillPointLoadoutsScreen;
+import com.wynntils.utils.mc.McUtils;
+import net.minecraft.client.gui.screens.inventory.ContainerScreen;
+import net.minecraft.network.chat.Component;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.UTILITIES)
+public class SkillPointLoadoutsFeature extends Feature {
+    @SubscribeEvent
+    public void onCharacterInfoScreenOpened(ScreenOpenedEvent.Post e) {
+        if (!(e.getScreen() instanceof ContainerScreen screen)) return;
+        if (!Models.Container.isCharacterInfoScreen(screen)) return;
+
+        screen.addRenderableWidget(
+                new LoadoutScreenButton(screen.width / 2 - LoadoutScreenButton.BUTTON_WIDTH / 2, screen.topPos - 24));
+    }
+
+    private static final class LoadoutScreenButton extends WynntilsButton {
```

Comment:
Missed this button too.

Also, do these buttons need to be separate classes, I think you can just use the button builder as from what I can tell none of the buttons do anything different from a regular button

Reply:
This button specifically can be in the feature class, as it's injected by the feature, and not used on a custom screen. 

Do you want to use button builders in regular screens? Sure, we did that before, if there is nothing special about them.

Reply:
I think using the builder is fine, for example the load button can just be this and (I think) it would work exactly the same, no need for a seperate LoadButton class

```
loadButton = new Button.Builder(
                        Component.translatable("screens.wynntils.skillPointLoadouts.load"),
                        (b) -> Models.SkillPoint.loadLoadout(selectedLoadout.key()))
                .pos((int) (dividedWidth * 35), (int) (dividedHeight * 52))
                .size((int) ((dividedWidth * 44) - (dividedWidth * 35)), BUTTON_HEIGHT)
                .build();
```

Reply:
Hm, it very well could be changed, it is a lot less lines, so maybe it's cleaner. Although having a separate class has it's cleanness too. Let's leave it up to DonkeyBlaster to decide.

Reply:
I will change it to a builder, having a separate class in a non-screen feels like a waste